pkg/api/v1alpha1/tinkermachineconfig.go

```diff
@@ -9,7 +9,115 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const TinkerbellMachineConfigKind = "TinkerbellMachineConfig"
+const (
+	TinkerbellMachineConfigKind = "TinkerbellMachineConfig"
+	TinkerbellDefaultTemplate   = `version: "0.1"
```

Comment:
Add TODO to save this as a struct instead of string
```suggestion
        // TODO: convert this into struct
	TinkerbellDefaultTemplate   = `version: "0.1"
```